universum/modules/output/output.py

```diff
@@ -22,15 +22,24 @@ class Output(Module):
     terminal_driver_factory = Dependency(TerminalBasedOutput)
     html_driver_factory = Dependency(HtmlOutput)
 
+    html_log_default_name = "universum_log"
```

Comment:
I think it would be better to keep the default log file name in the HtmlOutput class.
In case user didn't supply the name, the instance of the HtmlOutput should be created without any parameter.
They it should be able to decide what to do on its side.